snooty.toml

```diff
@@ -97,3 +97,16 @@ tls = ":abbr:`TLS (Transport Layer Security)`"
 url = ":abbr:`URL (Uniform Resource Locator)`"
 utc = ":abbr:`UTC (Coordinated Universal Time)`"
 vpc = ":abbr:`VPC (Virtual Private Cloud)`"
+
+[[banners]]
+targets = ["command/atlas-dataLakes*.txt"]
+variant = "info"
+value = """\
+    ``atlas dataLake`` commands control the functionality previously \ 
+    named Data Lake, which is now called "Data Federation" in the \ 
```

Comment:
**Suggestion:** I would add a link to the data federation docs because this banner has the potential to be confusing. Change the link if the URL will change after MDBW. Also added a comma
```suggestion
    named Data Lake, which is now called `Data Federation <https://www.mongodb.com/docs/datalake/>`__, in the \ 
```